_posts/2018-12-03-commons-clause.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Commons Clause at Zalando
+date: 2018-12-03
+author: The Open Source Team
+---
+
+# Introduction
+- What is the CC license?
+- Where/why was it created?
+
+# The Spirit of Open Source
+- The importance of Freedom 0
+- When you commit to Open Source, you commit to your community
+- Commons Clause breaks this covenant
```

Comment:
Mongo and Redis have benefitted from open source and community contributions - and now they cash in - would anyone have contributed to this if the commons clause was in place before..